From 6eaf6b1a61321acb824f9672c12d0e30a2110f52 Mon Sep 17 00:00:00 2001
From: Ong Boon Leong <boon.leong.ong@intel.com>
Date: Mon, 16 Mar 2015 11:10:43 +0800
Subject: [PATCH] DPDK: defconfig: add selectable RTE_MACHINE option

In order to make sure that DPDK build environment config can handle the
package being built on different build machine for different target machine
which will have different x86 architecture intrinsics than the build machine,
we introduce DPDK_TARGET_MACH to be tunable from dpdk recipe. The machine
type can be under mk/machine folder.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Ong Boon Leong <boon.leong.ong@intel.com>
---
 config/defconfig_i686-native-linuxapp-gcc   |    2 +-
 config/defconfig_x86_64-native-linuxapp-gcc |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/config/defconfig_i686-native-linuxapp-gcc b/config/defconfig_i686-native-linuxapp-gcc
index a90de9b..c9a74a5 100644
--- a/config/defconfig_i686-native-linuxapp-gcc
+++ b/config/defconfig_i686-native-linuxapp-gcc
@@ -32,7 +32,7 @@
 
 #include "common_linuxapp"
 
-CONFIG_RTE_MACHINE="native"
+CONFIG_RTE_MACHINE=$(DPDK_TARGET_MACH)
 
 CONFIG_RTE_ARCH="i686"
 CONFIG_RTE_ARCH_I686=y
diff --git a/config/defconfig_x86_64-native-linuxapp-gcc b/config/defconfig_x86_64-native-linuxapp-gcc
index 60baf5b..0e4802d 100644
--- a/config/defconfig_x86_64-native-linuxapp-gcc
+++ b/config/defconfig_x86_64-native-linuxapp-gcc
@@ -32,7 +32,7 @@
 
 #include "common_linuxapp"
 
-CONFIG_RTE_MACHINE="native"
+CONFIG_RTE_MACHINE=$(DPDK_TARGET_MACH)
 
 CONFIG_RTE_ARCH="x86_64"
 CONFIG_RTE_ARCH_X86_64=y
-- 
1.7.9.5

